src/components/People/People.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  FC, useEffect, useState,
+} from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person as PersonType } from '../../types';
+import { Person } from '../Person/Person';
+
+export const People: FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [peoples, setPeople] = useState<PersonType[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const findMotherSlug = (person: PersonType) => {
+    const result = peoples.find(item => item.name === person.motherName);
+
+    return result?.slug;
+  };
+
+  const findFatherSlug = (person: PersonType) => {
+    const result = peoples.find(item => item.name === person.fatherName);
+
+    return result?.slug;
+  };
```

Comment:
You do almost the same, so it will be better to create one function instead of these, `findPersonByName`.